locust/web.py

```diff
@@ -401,4 +411,57 @@ def _sort_stats(stats):
 def transform(text_file_contents):
     return text_file_contents.replace("=", ",")
 
-
+def send_opsgenie_request(_message):
+    s = requests.Session()
+    header =    {
+                    "Authorization": "GenieKey 517f286e-5d4e-4e3b-82d9-8e796e0ef4ba",
+                    "Content-type": "application/json"
+                }
+    data = {
+                "time": {
+                    "type" : str(_message)
+                },
+                "rules": [
+                    {
+                        "entity": {
+                            "id": "333016a8-71f3-444b-89c9-b170353162ba",
+                            "type": "integration"
+                        }
+                    },
+                    {
+                        "entity": {
+                            "id": "c3975c97-eea0-4816-9582-c7c6ffdbcd3a",
+                            "type": "integration"
+                        }
+                    },
+                    {
+                        "entity": {
+                            "id": "71f16a3c-0c4c-4404-b6b9-a787ede12885",
+                            "type": "integration"
+                        }
+                    }
+                ]
+            }
+
+    if _message != "for-1-hour":
+        startTime = datetime.utcnow()
+        endTime = startTime + timedelta(minutes = 15)
+        data['time']['startDate'] = startTime.strftime("%Y-%m-%dT%H:%M:%SZ")
+        data['time']['endDate'] = endTime.strftime("%Y-%m-%dT%H:%M:%SZ")
+
+    url = "https://api.opsgenie.com/v1/maintenance"
+    r = s.post(url, headers=header, data=json.dumps(data))
+    response = r.json()
+    global opsgenie_id
+    opsgenie_id = response['data']['id']
+
+def send_opsgenie_delete():
+    s = requests.Session()
+    header =    {
+                    "Authorization": "GenieKey 517f286e-5d4e-4e3b-82d9-8e796e0ef4ba",
+                    "Content-type": "application/json"
+                }
+
+    global opsgenie_id
+    url = "https://api.opsgenie.com/v1/maintenance/" + opsgenie_id + "/cancel"
```

Comment:
opsgenie_id is null when I unchecked the checkbox, but it causing this line error since opsgenie_id isn't string